File: plusMinus.rb
#!/bin/ruby

require 'json'
require 'stringio'

#
# Complete the 'plusMinus' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.

def plusMinus(arr)
  # Write your code here
  count_zero = 0
  count_positive = 0
  count_negative = 0
  (0..arr.length).each do |i|
    if (arr[i]).zero?
      count_zero += 1
    elsif (arr[i]).positive?
      count_positive += 1
    elsif (arr[i]).negative?
      count_negative += 1
    end

    puts count_positive/arr.length
    puts count_negative/arr.length
    puts count_zero/arr.length

  end
end
n = gets.strip.to_i
arr = gets.rstrip.split.map(&:to_i)

plusMinus arr

File: plusMinus.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'plusMinus' function below.
#
# The function accepts INTEGER_ARRAY arr as parameter.
#

def plusMinus(arr):
    count_p = 0
    count_n = 0
    count_z = 0

    for i in range(0,len(arr)):
        if arr[i] > 0:
            count_p = count_p + 1
        if arr[i] < 0:
            count_n = count_n + 1
        if arr[i] == 0:
            count_z = count_z + 1
    
    array_length = len(arr)
    # faltou o round aqui
    print(count_p/array_length)
    print(count_n/array_length)
    print(count_z/array_length)
    

if __name__ == '__main__':
    n = int(input().strip())

    arr = list(map(int, input().rstrip().split()))

    plusMinus(arr)
